master/buildbot/process/builder.py

```diff
@@ -673,7 +673,7 @@ def getMergeRequestsFn(self):
 
         # then translate False and True properly
         if mergeRequests_fn is False:
-            mergeRequests_fn = None
+            mergeRequests_fn = Builder._skipMergeRequestFn
```

Comment:
Right, this looks like we can pass there anything, not just a function. I suppose that there is no validation on the value type. Also, if the `mergeRequest` is a function already - we could skip all the `if `'s further logic.

Does it mean also if we have merge enabled for the particular builder we can't disable it through the global config?